app/assets/stylesheets/blocks/_font_awesomes.scss

```diff
@@ -26,7 +26,9 @@
 a {
    i {
      &.fas {
-
+       &.fa-question-circle{
+        margin-left: 1em;
+       }
```

Comment:
I think the fa-question-circle was already used somewhere else, does this also affect how it is used there?

Reply:
I went through whole project and found two places only that use `fa-question-circle` class. One is _links.html.erb and and one is _popover.html.erb. It used to have `&nbsp;` to hard-coding a white space before the icon
![Screen Shot 2021-12-09 at 13 39 51](https://user-images.githubusercontent.com/92752107/145456275-4666a65d-f44c-4679-9d75-4f6138cbf987.png)

To me this is not a good practice. `&nbsp; ` should not be used from style level to create whitespace (use `margin` in in `sass` instead, so we will have a unified styling experience for all such icons, as well as easier to maintain code in the future). We can use `&nbsp; `  in text though for editing paragraph content. I personally support to make styles in sass for all styling requirments. 

Thus I remove the `&nbsp;` for the existing `fa-question-circle` icons (only two) and add the margin-left instead. Please let me know what you think :)

Reply:
Other &nbsp; are still there. I didn't change it in case it will affect other features. I just change this question mark icon

Reply:
I do agree that using `margin` in `sass` would be a better approach to this. However, a quick search on the code shows that `&nbsp;` is used widely in the views; with this in mind, we should keep them as is to avoid veering from the main code.

Reply:
Wouldn't this only affect `fa-question-circle` class since the style is only in `&.fa-question-circle`? we only have _links.html.erb and and one is _popover.html.erb using this class.

Reply:
If you take at some views you will see that it is also used for spacing. An example is `app/views/org_admin/templates/_row.html.erb`.

I do agree that it is not best practice but we should keep it as is to avoid divergence unless it becomes a problem or a change in feature that we want to make. Following that train of thought, we should avoid doing code cleanup unless it has to do with code we wrote ourselves.

Reply:
This is to avoid a long list of overlapping code changes that need to be fixed when merging their updates.

Hope this makes sense.

Reply:
Got it. It make sense to use their convention to avoid merging issue in the future. I change my code to use nbsp also. I guess we can let them know in our next meeting to suggest them for a better coding practice